plugins/modules/vyos_firewall_global.py

```diff
@@ -430,30 +428,22 @@
 #
 #
 - name: Render the commands for provided  configuration
-  vyos_firewall_global:
-    running_config:
-      "set firewall all-ping 'enable'
-  set firewall broadcast-ping 'enable'
-  set firewall config-trap 'enable'
-  set firewall group address-group ENG-HOSTS address '192.0.3.1'
-  set firewall group address-group ENG-HOSTS address '192.0.3.2'
-  set firewall group address-group ENG-HOSTS description 'Sales office hosts address list'
-  set firewall group address-group SALES-HOSTS address '192.0.2.1'
-  set firewall group address-group SALES-HOSTS address '192.0.2.2'
-  set firewall group address-group SALES-HOSTS address '192.0.2.3'
-  set firewall group address-group SALES-HOSTS description 'Sales office hosts address list'
-  set firewall group network-group MGMT description 'This group has the Management network addresses'
-  set firewall group network-group MGMT network '192.0.1.0/24'
-  set firewall ip-src-route 'enable'
-  set firewall log-martians 'enable'
-  set firewall receive-redirects 'disable'
-  set firewall send-redirects 'enable'
-  set firewall source-validation 'strict'
-  set firewall state-policy established action 'accept'
-  set firewall state-policy established log 'enable'
-  set firewall state-policy invalid action 'reject'
-  set firewall syn-cookies 'enable'
-  set firewall twa-hazards-protection 'enable'"
+  vyos.vyos.vyos_firewall_global:
+    running_config: set firewall all-ping 'enable' set firewall broadcast-ping 'enable'
```

Comment:
This is not readable anymore. Can you see if running black (19.3b0) with line-length 79 fixes it?